packages/cozy-harvest-lib/src/components/hooks/useAppLinkWithStoreFallback.jsx

```diff
@@ -0,0 +1,41 @@
+import { useEffect, useState } from 'react'
+import { Q, models } from 'cozy-client'
+
+const { applications: ApplicationsModel } = models
```

Comment:
Why do we rename this import to ApplicationsModel ?
It seems like ApplicationsModel is a constructor when it is only a namespace for helper functions.

Not in the scope of this PR: I have the feeling that applications/jobs/triggers/files namespaces for helpers could directly be exported from `cozy-client` and not from a models object. It would enable more precise tree shaking (here we do not need all the models, only the applications).